# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../raidtools/largefile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We have large file support these days and the custom syscall does
often not build anymore.

--- raidtools-1.00.3/raid_io.c.vanilla	2007-05-12 11:37:47.594872668 +0200
+++ raidtools-1.00.3/raid_io.c	2007-05-12 11:39:52.325980688 +0200
@@ -81,7 +81,7 @@
 }
 #undef F
 
-#if !(defined(__alpha__) || defined(__sparc_v9__) || defined(__ia64__))
+#if 0
 # ifndef __NR__llseek
 #  ifdef __sparc__
 #   define __NR__llseek		236
@@ -105,7 +105,7 @@
 	long long result;
 	int retval;
 
-#if defined(__alpha__) || defined(__sparc_v9__) || defined(__ia64__)
+#if 1
 	return lseek(fd, offset, SEEK_SET);
 #else
 	retval = _llseek (fd, ((unsigned long long) offset) >> 32,
